run_cart.sh

```diff
@@ -54,7 +55,7 @@ function wait {
 # run_server np cmd
 function run_bg {
         echo "run_bg CMD:"
-        cmd="orterun --mca btl tcp,self --mca oob tcp --mca mtl ^psm2,ofi --mca pml ob1 --map-by node --timeout $timeout --np $1 --hostfile $2 --output-filename testLogs/$4/sep_$test_$sep -x D_LOG_FILE=testLogs/$4/sep_$test_$sep/srv_output.log -x D_LOG_MASK=ERR -x CRT_PHY_ADDR_STR=\"ofi+verbs;ofi_rxm\" -x OFI_INTERFACE=$interface -x OFI_DOMAIN=mlx5_0  -x CRT_CTX_SHARE_ADDR=$sep -x CRT_CTX_NUM=16 -x PATH -x LD_LIBRARY_PATH $3 &"
+        cmd="orterun --mca btl tcp,self --mca oob tcp --mca mtl ^psm2,ofi --mca pml ob1 --map-by node --timeout $timeout --np $1 --hostfile $2 --output-filename testLogs/$4/sep_$test_$sep -x D_LOG_FILE=testLogs/$4/sep_$test_$sep/srv_output.log -x D_LOG_MASK=ERR -x CRT_PHY_ADDR_STR=\"ofi+verbs;ofi_rxm\" -x OFI_INTERFACE=$interface -x OFI_DOMAIN=mlx5_0  -x CRT_CTX_SHARE_ADDR=$sep_cart -x CRT_CTX_NUM=16 -x PATH -x LD_LIBRARY_PATH $3 &"
```

Comment:
Are the --mca params here still needed for running with verbs?

Reply:
I have not added those I just changed the log folder variable. I recall there were few errors when you do not use some of the variables.